worldedit-core/src/main/java/com/sk89q/worldedit/command/SchematicCommands.java

```diff
@@ -313,9 +313,9 @@ public void load(
             Actor actor, LocalSession session,
             @Arg(desc = "File name.")
                     String filename,
-            @Arg(desc = "Format name.", def = "fast")
-                    String formatName,
             //FAWE start - random rotation
+            @Arg(desc = "Format name.", def = "")
+                    String formatName,
```

Comment:
Why this change?

Reply:
Otherwise we don't know whether the user wrote `//schem load <name>` or `//schem load <name> fast`, but those cases need to be handled separately, otherwise it's impossible to to use e.g. `//schem load <name> slow`.